packages/design/src/icons/iconMap.ts

```diff
@@ -439,9 +450,6 @@ export const iconMap = {
     ],
     "today": [
       "M682.667 128c-23.565 0-42.667 19.103-42.667 42.667v42.667h-256v-42.667c0-23.564-19.103-42.667-42.667-42.667s-42.667 19.103-42.667 42.667v42.667h-85.333c-47.128 0-85.333 38.205-85.333 85.333v512c0 47.13 38.205 85.333 85.333 85.333h597.333c47.13 0 85.333-38.204 85.333-85.333v-512c0-47.128-38.204-85.333-85.333-85.333h-85.333v-42.667c0-23.564-19.102-42.667-42.667-42.667zM810.667 298.667v85.333h-597.333v-85.333h597.333zM213.333 469.333h597.333v341.333h-597.333v-341.333zM704 640c-35.345 0-64-28.655-64-64s28.655-64 64-64c35.345 0 64 28.655 64 64s-28.655 64-64 64z"
-    ],
-    "drag": [
```

Comment:
This file was autogenerated, is this change expected? 🤔

Reply:
Yup this is ok - somehow this was added to the bottom of `iconMap.ts` when it should have been at the top.  Probably related to how it was exported at the time.
Either way your PR fixed that for us, so thank you 👍